modules/sdk-api/src/bitgoAPI.ts

```diff
@@ -748,10 +750,44 @@ export class BitGoAPI implements BitGoBase {
     this._token = accessToken;
   }
 
+  public createAccountKeychain = async (password: string): Promise<any> => {
+    const keyData = this.keychains().create();
+    const hdNode = bitcoin.HDNode.fromBase58(keyData.xprv);
+
+    return await this.keychains().add({
+      source: 'ecdh',
+      xpub: hdNode.neutered().toBase58(),
+      encryptedXprv: this.encrypt({
+        password: password,
+        input: hdNode.toBase58(),
+      }),
+    });
+  };
+
+  private updateUserSettings(params: any): Promise<any> {
+    return this.put(this.url('/user/settings', 2)).send(params).result();
+  }
+
+  private ensureUserEcdhKeychainIsCreated = async (password: string): Promise<any> => {
```

Comment:
```suggestion
  private ensureUserEcdhKeychainIsCreated = async (loginPassword: string): Promise<any> => {
```

we use password as a name for wallets as well, its nice to be explicit.